<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PeelingClass</class>
 <widget class="QWidget" name="PeelingClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>411</width>
    <height>344</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Peeling</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="styleSheet">
      <string notr="true">QTabBar::tab { 
	background-color: rgb(255, 255, 255);
	font: 6pt &quot;宋体&quot;;
	border: none;
	margin-right: 20px;
	margin-left: 2px;
	min-width: 100px;
	min-height: 40px;
	padding-bottom: -1px;
}
QTabBar::tab:selected { 
	border-bottom: 5px solid red;
	font-weight: bold;
	font: 14pt &quot;黑体&quot;;
}
QTabWidget::pane {
	border: none;
}</string>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>主题</string>
      </attribute>
      <widget class="QPushButton" name="btn_black">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>115</width>
         <height>115</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(0, 0, 0);
border-radius:10px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../../res/换肤/black.png</normaloff>../../res/换肤/black.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>115</width>
         <height>115</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_red">
       <property name="geometry">
        <rect>
         <x>145</x>
         <y>10</y>
         <width>115</width>
         <height>115</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius:10px;
background-color: rgb(236, 65, 65);</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../../res/换肤/red.png</normaloff>../../res/换肤/red.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>115</width>
         <height>115</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_pink">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>10</y>
         <width>115</width>
         <height>115</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(236, 147, 184);
border-radius:10px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../../res/换肤/pink.png</normaloff>../../res/换肤/pink.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>115</width>
         <height>115</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_blue">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>150</y>
         <width>115</width>
         <height>115</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius:10px;
background-color: rgb(135, 216, 254);</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../../res/换肤/blue.png</normaloff>../../res/换肤/blue.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>115</width>
         <height>115</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_green">
       <property name="geometry">
        <rect>
         <x>145</x>
         <y>150</y>
         <width>115</width>
         <height>115</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius:10px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../../res/换肤/green.png</normaloff>../../res/换肤/green.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>115</width>
         <height>115</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_gold">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>150</y>
         <width>115</width>
         <height>115</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius:10px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../../res/换肤/gold.png</normaloff>../../res/换肤/gold.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>115</width>
         <height>115</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
